react-native-in-app-payments-quickstart/android/app/src/main/res/values/themes.xml

```diff
@@ -8,7 +8,7 @@
     </style>
 
     <style name="sqip.Theme.CardEntry" parent="AppTheme">
-        <item name="sqipSaveButtonText">@string/pay_button</item>
+<!--        <item name="sqipSaveButtonText">@string/pay_button</item>-->
```

Comment:
Why do we comment this line?

Reply:
Because after plugin upgration it's not supported.

Reply:
Shall we just remove it?

Reply:
Removed it.